app/models/event.rb

```diff
@@ -0,0 +1,8 @@
+class Event < ApplicationRecord
```

Comment:
- [ ] Add validations to the event fields, title, body, etc, not allowing empty events, making sure there's a minimum and/or maximum length, etc. Check the [validations documentation here](https://guides.rubyonrails.org/active_record_validations.html).